client/src/plugins/report-feedback/ReportFeedback.js

```diff
@@ -100,7 +100,7 @@ export class ReportFeedback extends PureComponent {
 
     return (
       <Fragment>
-        <Fill slot="status-bar__app" group="9_feedback">
+        <Fill slot="status-bar__app" group="8_feedback">
```

Comment:
What is the reason for this change? It moves buttons around (on the status bar).

Reply:
To accomodate the new `Update` button